backend/uclapi/dashboard/tests.py

```diff
@@ -298,7 +298,7 @@ def test_edit_webhook_POST_ownership_verification_fail(
             content["message"],
             "Ownership of webhook can't be verified."
             "Make sure to follow the documentation: "
-            "https://docs.uclapi.com/#challenge-event"
+            "https://uclapi.com/docs#webhook/challenge-event"
```

Comment:
I think this URL is missing a slash after `docs`

Reply:
The missing slash is intentional as I'm routing to a local element on the page by setting the href to the id of the DOM element. This is how the Material-UI docs work and what I felt was the easiest way to do it.